pkg/migrations/mls/20240109001927_add-messages.up.sql

```diff
@@ -0,0 +1,27 @@
+SET statement_timeout = 0;
+
+--bun:split
+
+CREATE TABLE group_messages (
+    id BIGSERIAL PRIMARY KEY,
+    created_at TIMESTAMP NOT NULL DEFAULT NOW(),
+    group_id VARCHAR(200) NOT NULL,
+    data BYTEA
```

Comment:
One thing we are going to want on this API is idempotency, like we have on the V1 publish APIs. If we had a unique index on a field that represented `HASH(CONCAT(group_id, data))` we could safely retry requests without worrying about duplicate messages.

Reply:
We actually have (topic, timestamp, content) uniqueness in v1, not (topic, content), so you can technically send the same content at a different time and it wouldn't be a duplicate. If there's no timestamp embedded in `data` then it would mean that you can't send the same message more than once if we do (group_id, data).

Reply:
You really shouldn't be able to construct two identical messages with MLS. There's always nonces and salts somewhere in there, changing up the encryption.